src/applications/vaos/containers/TypeOfCarePage.jsx

```diff
@@ -38,9 +42,13 @@ const pageTitle = 'Choose the type of care you need';
 
 export class TypeOfCarePage extends React.Component {
   componentDidMount() {
-    this.props.openTypeOfCarePage(pageKey, uiSchema, initialSchema);
-    document.title = `${pageTitle} | Veterans Affairs`;
-    scrollAndFocus();
+    if (this.props.isCernerOnlyPatient) {
+      this.props.router.replace('/');
```

Comment:
Should we put this in NewAppointmentLayout? May not be necessary since if the user were to bookmark any other URL in the flow, it would redirect here, and then redirect to `/` anyway. Just a thought.

Reply:
It probably makes more sense there, I was just trying to avoid connecting it to Redux. But I should just go ahead and do that.